circle/functions

```diff
@@ -174,6 +182,12 @@ get_target_branch() {
   echo "${result[0]}"
 }
 
+is_base_image() {
```

Comment:
We could consider adding a `IS_BASE_IMAGE=1` to the base images `circle.yml` files.
If you agree, you could add support for that here:
```
[[ "${IS_BASE_IMAGE}" == "1" ]] || [[ "${image_name}...
```